src/System.Private.Reflection.Metadata.Ecma335/src/Permanent/StrongBox.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Holds a reference to a value.
+    /// </summary>
+    /// <typeparam name="T">The type of the value that the <see cref = "StrongBox{T}"></see> references.</typeparam>
+    internal class StrongBox<T>
```

Comment:
This can be moved to CoreLib without problems as well.

There is *very little* left in https://github.com/dotnet/corefx/blob/master/src/System.Runtime/src/System.Runtime.csproj#L32 - it may make sense to move everything that's left in System.Runtime facade to corelib.